app/models/manageiq/providers/cloud_manager/placement_group.rb

```diff
@@ -0,0 +1,74 @@
+class ManageIQ::Providers::CloudManager::PlacementGroup < ::PlacementGroup
+  acts_as_miq_taggable
+  has_and_belongs_to_many :vms, :join_table => :placement_group_vms, :foreign_key => :placement_group_id
```

Comment:
I thought we said a VM could only be in one placement_group?  If so you don't need a habtm a simple has_many should do